main/test/ca/uwaterloo/flix/language/feature/Test.Expression.Int.flix

```diff
@@ -25,3 +25,6 @@ def testInt07(): Int = 2147483647
 
 @test
 def testInt08(): Int = -2147483648
+
+@test
+def testInt09(): Int = 1_0
```

Comment:
I'd like to see a handful of test cases for each type. For Int32, for example, I would expect to see:

```
1_000
42_000
1_000_000
42_000_000
1_000_000_000
42_000_000_000
```

or the like. Just to make sure we cover more ground.